airflow/config_templates/config.yml

```diff
@@ -2677,7 +2669,17 @@ dag_bundles:
     Configuration for the DAG bundles. This allows Airflow to load DAGs from different sources.
 
   options:
-    backends:
+    dag_bundle_storage_path:
```

Comment:
You missed the move where this option is used, and tests.

Reply:
lol